docs/manual/java/gettingstarted/lagom-examples.md

```diff
@@ -0,0 +1,19 @@
+# Learning more from Lagom examples
+
+After getting started with Hello World, you can learn more about Lagom by downloading and running one of the examples from GitHub:
+
+* The [Chirper](https://github.com/search?utf8=%E2%9C%93&q=Lagom+chirper) template demonstrates a Twitter-like application and is available for Java and Scala, with Java variations that demonstrate use of JPA and JDBC.
+* The [Online Auction](https://github.com/search?utf8=%E2%9C%93&q=lagom%2Fonline+auction&type=Repositories&ref=searchresults) demonstrates an eBay-like application and is available for both Java and Scala.
+
+The following table outlines some of the Lagom features you can observe in each of the examples.
+
+Feature |Chirper| Online Auction |
+--------|:--------:|:-------------:|
+How to use persistence| Y | Y 
+How to use the message broker API| N | Y
+How to communicate between services| Y | Y
+How streaming service calls work | Y | N
+How to connect a front-end Play app | Y | Y
+
```

Comment:
I've changed as you suggested, with a minor change:

How to develop a Play app with Lagom
How to consume Lagom services from a front-end Play app

Not sure about "Play app with Lagom", but "on" Lagom isn't quite the correct word.